extensions/amp-list/0.1/amp-list.js

```diff
@@ -602,6 +606,14 @@ export class AmpList extends AMP.BaseElement {
             'update': false,
           });
         }
+        toArray(
+          dev().assertElement(this.container_).children
+        ).forEach((child) =>
+          this.owners_./*OK*/ scheduleUnlayout(
+            dev().assertElement(this.container_),
```

Comment:
This must be the owning amp-element, not a regular `<div>`. So, the `<amp-list>`?